cmd/lncli/cmd_open_channel.go

```diff
@@ -845,13 +853,24 @@ func batchOpenChannel(ctx *cli.Context) error {
 		return nil
 	}
 
+	// Coin selection strategy options: "largest" or "random" only, as
+	// these are currently the supported strategies.
+	coinSelectionStrategy := ctx.String("coin_selection_strategy")
+	if coinSelectionStrategy != "largest" &&
+		coinSelectionStrategy != "random" {
```

Comment:
I am wondering if a switch statement would be better here, that way as coinSelection strategy grows we can just add it as a new case?